src/_style.scss

```
@@ -14,6 +14,22 @@ $microsoft-focus-black: #000;
 $apple-black: #000000;
 $apple-focus-black: $apple-black;
 
+.sr-only {
+  position:absolute;
+  left:-10000px;
+  top:auto;
+  width:1px;
+  height:1px;
+  overflow:hidden;
```

Can you add space between key value pairs

```suggestion
  overflow: hidden;
```